packages/apt/test.sh

```diff
@@ -148,6 +152,14 @@ if [ ${pg_regress_status} -ne 0 ]; then
   exit ${pg_regress_status}
 fi
 
+echo "::group::Run systemd timer test"
+
+$(pg_config --bindir)/createuser postgres --superuser
+/host/packages/test-systemd-timer.sh
```

Comment:
systemdのタイマーは実現方法であって、テストしたいものはprimary-maintainerなので、ファイル名を変えようか。

```suggestion
/host/packages/test-primary-maintainer.sh
```